checks.d/postgres.py

```diff
@@ -554,6 +554,7 @@ def check(self, instance):
         tags = instance.get('tags', [])
         dbname = instance.get('dbname', None)
         relations = instance.get('relations', [])
+        ssl = instance.get('ssl', False)
```

Comment:
You probably want to use `_is_affirmative` from the config module. Otherwise you could end up with some funky string parsing.
See https://github.com/DataDog/dd-agent/commit/0e862f5f7f16f5fc1466f4b8ebff1b4e44cac64e